docs/source/model-registry.rst

```diff
@@ -34,6 +34,10 @@ Model Alias
 
     Aliases are especially useful for deploying models. For example, you could assign a ``champion`` alias to the model version intended for production traffic and target this alias in production workloads. You can then update the model serving production traffic by reassigning the ``champion`` alias to a different model version.
 
+    .. note::
+
+        Model Alias is not case-sensitive. For example, ``champion`` and ``Champion`` are both converted to lowercase and considered the same alias.
```

Comment:
so this is databricks-only behavior?

Reply:
[It seems so](https://github.com/mlflow/mlflow/issues/12331#issuecomment-2164959769)